pootle/apps/pootle_store/form_fields.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django import forms
+from django.utils.datastructures import MergeDict, MultiValueDict
+
+from pootle.core.dateparse import parse_datetime
+from pootle_misc.checks import CATEGORY_IDS
+
+
+class CommaSeparatedCheckboxSelectMultiple(forms.CheckboxSelectMultiple):
+
+    def value_from_datadict(self, data, files, name):
+        # Accept `sfields` to be a comma-separated string of fields (#46)
+        if "," in data.get(name, ""):
+            return data.get(name).split(u',')
+        if isinstance(data, (MultiValueDict, MergeDict)):
+            return data.getlist(name)
+        return data.get(name, None)
+
+
+class MultipleValueWidget(forms.TextInput):
+
+    def value_from_datadict(self, data, files, name):
+        if hasattr(data, "getlist"):
+            return data.getlist(name)
+        return []
+
+
+class MultipleArgsField(forms.Field):
+    widget = MultipleValueWidget
+
+    def __init__(self, *la, **kwa):
```

Comment:
This has bothered me for some time.  There doesn't seem to be any good reason not to follow Python conventions on this.  Yes it just a convention, but then so is self and cls and we follow those to make it easier for anyone reading the code.